build.gradle

```diff
@@ -12,10 +12,16 @@ subprojects { subproject ->
     subproject.apply plugin: 'java-library'
     subproject.apply plugin: 'com.github.johnrengelman.shadow'
     subproject.apply plugin: 'maven-publish'
+    subproject.apply plugin: 'java'
```

Comment:
This can go too if the UTF-8 gradle config is not needed.

Reply:
Removed.